### ISSUE TITLE
GitHub dependencies _can_ be automatically installed

### ISSUE THREAD

**Issue by jimhester** [ecos-umons/SANER2016#1]
The paper repeatedly states the following statement

> GitHub packages cannot currently be installed automatically

However using the `Remotes:` feature of devtools you _can_ have development dependencies (such as those from GitHub) installed automatically. See https://github.com/hadley/devtools/blob/master/vignettes/dependencies.Rmd for more information on the syntax needed in the DESCRIPTION file.

See the current [devtools DESCRIPTION](https://github.com/hadley/devtools/blob/782e6b071d058eea53aae596a3c120d61df2f0b4/DESCRIPTION#L46), the [r4ds](https://github.com/hadley/devtools/blob/782e6b071d058eea53aae596a3c120d61df2f0b4/DESCRIPTION#L46) and [ggplot2](https://github.com/hadley/ggplot2-book) books and this [search](https://github.com/search?p=1&q=filename%3ADESCRIPTION+%22Remotes%3A%22&ref=searchresults&type=Code&utf8=%E2%9C%93) for other repositories using the feature in practice.

While this feature is relatively new (hadley/devtools#902), it is unfortunate that the authors were not aware of this before preparing this paper.

**Comment by phgrosjean** [comment]
You are right. Note that the material used in that paper was analysed _before_ that feature was introduced in devtools, and even the manuscript was in final form. None of us noticed this change during the review.
